extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -323,16 +323,22 @@ export class AmpAutocomplete extends AMP.BaseElement {
   /**
    * Create and return <div> element from given plan-text item.
    * @param {string} item
+   * @param {string=} substring
    * @return {!Element}
    * @private
    */
-  createElementFromItem_(item) {
+  createElementFromItem_(item, substring) {
     const element = this.element.ownerDocument.createElement('div');
     element.classList.add('i-amphtml-autocomplete-item');
     element.setAttribute('role', 'listitem');
     element.setAttribute('data-value', item);
     element.setAttribute('dir', 'auto');
-    element.textContent = item;
+    if (!substring || substring.length > item.length) {
+      element.textContent = item;
+    } else {
+      const regex = new RegExp(substring, 'gi');
```

Comment:
This is an easy way to replace all instances at once, but if the query string contains `.*` will it match those literals or the whole string?

Reply:
I didn't experience this problem in manual testing. This does remind me though that it doesn't make sense to replace all instances regardless of filter types (i.e. prefix) since that is a disingenuous representation of how some suggestions are made.